src/components/DatePicker/index.stories.tsx

```diff
@@ -104,6 +125,24 @@ export const WithTitle = () => {
 
 WithTitle.storyName = 'With title'
 
+export const WithResetButton = () => {
```

Comment:
**suggestion (if-minor):** Would be cool to have a test for this as well. e.g. provide an initial date, search for the reset button and click it, check the date is no longer in the screen.